docs/Components/Powertrain/SimpleMotor.md

```diff
@@ -0,0 +1,57 @@
+# Simple Motor
+
+Simple Motor class is based on the class called "attatched component", and written over the Motor class. For the fact that it is an attatched component, it is needed to set a parent for it, and just like on attached component, all information from the parent will be taken in count on the children methods.
+
+Besides that the class needs some inputs values from the user, such as:
+* static_thrust
+* linear_coefficient
+* distance_origin_to_propeller
+
+## *Thrust Center* _method_:
+
+It's a property that returns a Vector2 with the values of the points where the thrust is applied.
+
+As it is a Simple Motor, by default the force is applied on x in the distance from the origin to the propeller and in y, in 0 as it's seen in the example below:
```

Comment:
As it is a simplified model of a motor, by default the force is applied on the motor axis (y = 0), on a given distance (distance_origin_to_propeller) over the horizontal X-axis, as it's seen in the example below: